scripts/cc-cli/src/commands/rotateKeys.ts

```diff
@@ -0,0 +1,17 @@
+import { Command, OptionValues } from "commander";
+import { newApi } from "../api";
+
+export function makeRotateKeysCommand() {
+  const cmd = new Command("rotate-keys");
+  cmd.option("-u, --url [url]", "URL for the Substrate node");
```

Comment:
```suggestion
  cmd.option("-u, --url [url]", "URL for the Substrate node", "ws://localhost:9944");
```
Since often this is run against a local node (especially because by default validator nodes _only_ allow this RPC to be called from localhost, since it isn't safe to expose externally)